### ISSUE TITLE
Raft (Integrated Storage) follower joined as non voters

### ISSUE THREAD

**Issue by LoboHacks** [hashicorp/vault#11177]
In the current 1.7.0 rc1 and in rc2 the raft followers are joined as non voters in a ha only setup.

This can be reproduced by running the hashicorp learn example:
[https://learn.hashicorp.com/tutorials/vault/raft-ha-storage](https://learn.hashicorp.com/tutorials/vault/raft-ha-storage)

```bash
VAULT_ADDR=http://127.0.0.1:8210 vault operator raft list-peers
Node       Address           State       Voter
----       -------           -----       -----
vault_1    127.0.0.1:8211    leader      true
vault_2    127.0.0.1:8221    follower    false # <---
```

**Comment by ncabatoff** [comment]
Hi @LoboHacks,

Thanks so much for doing testing of the RC!  Unfortunately it's too late to get a fix for this issue into the official release, but we'll address it in the following point release.

Until then, you can work around this issue by setting the environment variable VAULT_RAFT_AUTOPILOT_DISABLE=1 for your Vault servers.  Please note that this feature toggle may go away in 1.8, although by then it won't be necessary.

**Comment by holtwilkins** [comment]
Hey @ncabatoff , I'm a bit confused on the status here - basically raft HA storage autopilot is broken in 1.7.0, and was it fixed in 1.7.1 or is it still broken, there are just tests for it now?

**Comment by ncabatoff** [comment]
Hi @holtwilkins,

It's fixed in 1.7.1, by #11340.  I'm not sure why the changelog doesn't reflect that but I'll sort that out soon.

**Comment by ncabatoff** [comment]
Much later, I finally updated the changelog, closing.
